### ISSUE TITLE
hey , wanna make friends with you~

### ISSUE THREAD

**Issue by EliuFrank** [QiongWang1/Hello-word#2]

**Comment by QiongWang1** [comment]
Hey, nice to meet you !

**Comment by EliuFrank** [comment]
maybe here is the secret base?

**Comment by QiongWang1** [comment]
maybe, I guess

**Comment by EliuFrank** [comment]
so where are you ?  overseas student?

**Comment by QiongWang1** [comment]
I am an international student, and I come from Boston. What about you?

**Comment by EliuFrank** [comment]
chinese local student, in nanjing.  Are you chinese?

**Comment by QiongWang1** [comment]
Yes, I am Chinese, but I haven't travel to nanjing before, hahaha, so amazing connecting!

**Comment by EliuFrank** [comment]
hhhh, really nice to meet you ,today is the Beginning of Autumn, do you know the topic " the first cup of milk tea in autumn" ?  Have you drink milk tea today?

**Comment by EliuFrank** [comment]
oh , it should be "drunk" , not "drink" .🤭

**Comment by QiongWang1** [comment]
Yes, I know a little of this topic, but I have not opportunity to enjoy it, that's sad.

**Comment by EliuFrank** [comment]
That's all right , just a marketing method , maybe you can drink a cup of milk instead. Are you going to sleep?  we have 12-hour time difference.

**Comment by QiongWang1** [comment]
Yes, we have 12-hour time difference, today is still a Sunday for me.

**Comment by EliuFrank** [comment]
okay……good day, it’s a new week for me ,  extremely sleepy 🥱

---- Replied Message ----
| From | Qiong ***@***.***> |
| Date | 08/12/2024 06:12 |
| To | QiongWang1/Hello-word ***@***.***> |
| Cc | EliuFrank ***@***.***>,
Author ***@***.***> |
| Subject | Re: [QiongWang1/Hello-word] hey , wanna make friends with you~ (Issue #2) |

Yes, we have 12-hour time difference, today is still a Sunday for me.

**Comment by QiongWang1** [comment]
How's going in your new week? Are there any interesting things?

**Comment by EliuFrank** [comment]
I think it’s a sleepy week, I hate waking up every morning, by the way, I seem to have forgotten the posture of lying flat and sleeping, instead lying side/down and sleeping every night.

---- Replied Message ----
| From | Qiong ***@***.***> |
| Date | 08/15/2024 02:31 |
| To | QiongWang1/Hello-word ***@***.***> |
| Cc | EliuFrank ***@***.***>,
Author ***@***.***> |
| Subject | Re: [QiongWang1/Hello-word] hey , wanna make friends with you~ (Issue #2) |

How's going in your new week? Are there any interesting things?

**Comment by QiongWang1** [comment]
I always miss the messages from Git, why we don't talk by Whatsapp or WeChat? if you would like to

**Comment by EliuFrank** [comment]
oh yeah, i'd love to, but i only have wechat, you can send me your wechat QRcode? or the number?


| |
刘琦
|
|
***@***.***
|
---- Replied Message ----
| From | Qiong ***@***.***> |
| Date | 8/15/2024 08:39 |
| To | ***@***.***> |
| Cc | ***@***.***>,
***@***.***> |
| Subject | Re: [QiongWang1/Hello-word] hey , wanna make friends with you~ (Issue #2) |

I always miss the messages from Git, why we don't talk by Whatsapp or WeChat? if you would like to

**Comment by QiongWang1** [comment]
Base my WeChat set, It's better I add you, Do you mind sharing your WeChat?

**Comment by EliuFrank** [comment]
Sure , my wechat number is  **Dogricherliu**

**Comment by EliuFrank** [comment]
Sorry ,  it's  dogricherliu

**Comment by EliuFrank** [comment]
how's it going? long time to chat....
